ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -81,6 +81,23 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
         [self isAnonymousWithResult:result];
     } else if ([@"checkTrialOrIntroductoryPriceEligibility" isEqualToString:call.method]) {
         [self checkTrialOrIntroductoryPriceEligibility:arguments[@"productIdentifiers"] result:result];
+    } else if ([@"invalidatePurchaserInfoCache" isEqualToString:call.method]) {
+        [self invalidatePurchaserInfoCacheWithResult:result];
+    } else if ([@"setAttributes" isEqualToString:call.method]) {
+        NSDictionary *attributes = arguments[@"attributes"];
+        [self setAttributes:attributes result:result];
+    } else if ([@"setEmail" isEqualToString:call.method]) {
+        NSString *email = arguments[@"email"];
+        [self setEmail:email result:result];
+    } else if ([@"setPhoneNumber" isEqualToString:call.method]) {
+        NSString *phoneNumber = arguments[@"phoneNumber"];
+        [self setPhoneNumber:phoneNumber result:result];
+    } else if ([@"setDisplayName" isEqualToString:call.method]) {
+        NSString *displayName = arguments[@"displayName"];
+        [self setDisplayName:displayName result:result];
+    } else if ([@"setPushToken" isEqualToString:call.method]) {
+        NSString *pushToken = arguments[@"pushToken"];
+        [self setPushToken:pushToken result:result];
```

Comment:
I think by moving the arguments to the private methods, this will look better. I thought about doing that but thought it was not worth doing right now